pkg/networkservice/common/connect/server_test.go

```diff
@@ -1,636 +0,0 @@
-// Copyright (c) 2020-2021 Doc.ai and/or its affiliates.
-//
-// SPDX-License-Identifier: Apache-2.0
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at:
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package connect_test
```

Comment:
These tests cover many corner cases. Can we keep them?

Reply:
I totally agree with Denis, most of our `connect` tests have been motivated by some painful issues. Removing them will most probably lead us back to the old problems.

Reply:
@Bolodya1997 Added the tests back... and yes, they caught some issues :)

Reply:
I have added few missed `goleak` checks to `connect` tests (https://github.com/networkservicemesh/sdk/pull/1087), please can you rebase your tests?

Reply:
Done